### DISCUSSION TITLE
Is there a way to delete the "Follow" button?

### DISCUSSION THREAD

**Discussion by sam-lindquist** [academicpages/academicpages.github.io#1998] [Q&A] [answered]
I chose not to put any info (e.g. email, twitter, etc) below my bio and name, so when I click on the follow button it is blank. Is there a way that I can delete the follow button entirely? 

Thanks for any info this site is great!

**Comment by rjzupkoii** [comment]
This actually sounds like a bit of a bug since not having any social media should prevent that from rendering, I'm going to open up a new bug to fix it.

**Reply by rjzupkoii** [reply, marked as answer]
@sam-lindquist I just pushed an update to Issue #1999 which fixes this problem. If there is no social media then the "Follow" button will be removed when rendering the mobile view.

**Reply by sam-lindquist** [reply]
Thank you so much!!!! This website is great
